test/controllers/old_node_controller_test.rb

```diff
@@ -188,29 +188,31 @@ def test_current_version
   # authorised.
   def test_redact_node_unauthorised
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction))
     assert_response :unauthorized, "should need to be authenticated to redact."
   end
 
   ##
   # test the redaction of an old version of a node, while being
   # authorised as a normal user.
   def test_redact_node_normal_user
-    basic_authorization(users(:public_user).email, "test")
+    user = create(:user)
+    basic_authorization(user.email, "test")
 
     do_redact_node(nodes(:node_with_versions_v3),
-                   redactions(:example))
+                   create(:redaction, :user => user))
```

Comment:
I think it would be - the fact that redactions are shared across all moderators is a deliberate feature I believe and we have probably (accidentally) not really been testing that.

I've never really been clear why blocks behave the way they do - to my mind it would make more sense if any moderator could edit any block. Especially since I believe any moderator can revoke a block, they just can't edit it...